app/components/work_packages/activities_tab/journals/item_component.html.erb

```diff
@@ -1,14 +1,17 @@
 <%=
   component_wrapper(data: wrapper_data_attributes, class: "work-packages-activities-tab-journals-item-component") do
-    flex_layout(data: { 
+    flex_layout(data: {
       test_selector: "op-wp-journal-entry-#{journal.id}"
     }) do |journal_container|
       if show_comment_container?
         journal_container.with_row do
           render(border_box_container(
-                  id: "activity-anchor-#{journal.sequence_version}",
```

Comment:
For a moment I thought we used native browser anchor behaviour, but I guess that has never been the case.

So I am assuming that removing the ID here has no consequence.

Reply:
Good eye, the navigation was done by hand as it was reported to cause ["native scrolling issues"](https://github.com/opf/openproject/blob/dev/frontend/src/stimulus/controllers/dynamic/work-packages/activities-tab/index.controller.ts#L430) - in any case, since we need to support two formats, I moved the anchors to data attributes.